test/performance/README.md

```diff
@@ -111,13 +111,86 @@ To generate the sample workload for KubeStellar performance experiments proceed
     - 10 config maps containing a 2048-character random string
 
 
-
-
 5. Clean up the generated workload Kubernetes API objects from your cluster:
 
    ```bash
    $ cd $CL2_DIR
    $ ./cleanup.sh
    ```
 
-### Collect the creation & update timestamps for workload benchmark (OPTIONAL) (TO DO: WORK-IN-PROGRESS)
\ No newline at end of file
+
+### Collect the creation & update timestamps for workload benchmark (OPTIONAL)
+
+
+To collect the creation and status_update timestamps for the benchmark workload objects proceed as following:  
+
+1. Run the metrics collection script:
+
+   a) cd into the `test/performance` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash
+   $ cd .$HOME/kubestellar/test/performance
+   ```
+
+   b) Run the metrics collection script:
+
+   ```bash 
+   $ python3 metrics_collector.py <kubeconfig> <wds_ctx> <its_ctx> <wec_ctx> <num_ns> <output_dir> 
+   ```
+
+   For example:
+   
+   ```bash 
+   $ python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 2 $HOME/data
+   ```
+
+   Below is a detailed explanation of the input parameters:
+   - `kubeconfig`: path to the kubeconfig file, e.g., *$HOME/.kube/config*
+   - `wds_ctx`: name of the context for the target WDS, e.g., *"wds1"*
+   - `its_ctx`: name of the context for the target ITS, e.g., *"its1"*
+   - `wec_ctx`: name of the context for the target WEC, e.g., *"cluster1"*
+   - `num_ns`: number of namespaces created in your experiment, e.g., *2*
+   - `output_dir`: path to the directory for the output data files, e.g., *$HOME/data/*
+
+   For each namespace, the creation and status_update timestamps will be collected for the workload objects created in `wds1`, manifestwork & workstatus objects created in `its1`, and workload & appliedmanifestworks objects created in `wec1`, for example: let's assume that `num_ns=2`, then the following directories and files will be created: 
+
+   ```bash 
+      $tree
```

Comment:
What is `$tree`?

Reply:
It is a recursive directory listing program that produces a depth-indented listing of files -- source: https://www.geeksforgeeks.org/tree-command-unixlinux/

Reply:
The dollar sign was part of the question.
It looks like you are missing a space between the dollar sign and the command name?

Reply:
Fixed the space issue.